### PR TITLE
Create README for PF-1.1

### PR DESCRIPTION
**Pull request by sugrimov**
IPv4/IPv6 policy forwarding to indirect NH matching TZ-bits.

### CODE REVIEW

File: feature/experimental/policy_forwarding/match_dscp_tz_next_hop/README.md [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+Generate traffic on ATE Port 1 to test IPv4 and IPv6 destination networks with TZ=11 bits set. Generate flows for every DSCP value [3, 7, 11, 19, 27, 35, 39, 51, 55, 59].
+
+Verify:
+
+*  All traffic received on ATE Port 2.
+*  No packet loss when forwarding.
+*  Verify PF packet counters matching traffic generated.
+
+### PF-1.1.2: Verify PF no-match action
+Generate traffic on ATE Port 1 to test IPv4 and IPv6 destination networks with TZ=11 bits set. Generate flows for every DSCP value [0, 4, 8, 16, 24, 32, 36, 48, 52, 56].
```
Comment (daveruturaj) [inline feature/experimental/policy_forwarding/match_dscp_tz_next_hop/README.md]: Do you mean TZ bit not set?

Comment (sugrimov) [reply feature/experimental/policy_forwarding/match_dscp_tz_next_hop/README.md]: You have checked a previous commit.

File: feature/policy_forwarding/otg_tests/match_dscp_indirect_next_hop/README.md [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+
+### Configuration
+
+1.  All DUT Ports are configured as a singleton IP interfaces.
+
+2.  Static routes to test IPv4 and IPv6 destination networks are configured on DUT towards ATE Port 3.
+
+3.  eBGP session is configured on DUT port 2. Indirect /32 and /128 prefixes (PF next-hops) are announced via eBGP.
+
+4.  PF matching traffic marked with rightmost DSCP/TC 2 bits set to `11` is configured on DUT port 1. PF
```
Comment (daveruturaj) [inline feature/policy_forwarding/otg_tests/match_dscp_indirect_next_hop/README.md:33]: Can we rearrange the sentence for clarity?

PF is configured on DUT port 1 to match the traffic marked with  rightmost DSCP/TC 2 bits set to `11`.

File: feature/policy_forwarding/otg_tests/match_dscp_indirect_next_hop/README.md [hunk 3]
```diff
@@ -47,29 +51,43 @@ Verify:
...
+Generate traffic on ATE Port 1 to test IPv4 and IPv6 destination networks with DSCP/TC rightmost 2 bits set to `00`. Generate flows for every DSCP/TC values in the set [0, 8, 16, 24, 32, 40, 48, 56]. IPv6 flows should use TC 8-bit values [0, 32, 64, 96, 128, 160, 192, 224]
 
 Verify:
 
 *  All traffic received on ATE Port 3.
 *  No packet loss when forwarding.
 
 ### PF-1.1.3: Verify PF without NH present
-Withdraw next-hop prefixes from BGP announcement. Generate traffic on ATE Port 1 to test IPv4 and IPv6 destination networks with DSCP/TC rightmost 2 bits set to `11`. Generate flows for every DSCP/TC value in the set [3, 7, 11, 19, 27, 35, 39, 51, 55, 59]. 
+Withdraw next-hop prefixes from BGP announcement. Generate traffic on ATE Port 1 to test IPv4 and IPv6 destination networks with DSCP/TC rightmost 2 bits set to `11`. Generate flows for every DSCP/TC value in the set [3, 7, 11, 19, 27, 35, 39, 51, 55, 59].
```
Comment (daveruturaj) [inline feature/policy_forwarding/otg_tests/match_dscp_indirect_next_hop/README.md:62]: You didn't mention about IPc6 here. Is this intentional?

File: feature/policy_forwarding/otg_tests/match_dscp_indirect_next_hop/README.md [hunk 4]
```diff
@@ -0,0 +1,96 @@
...
+    ```
+                             |         | --eBGP-- | ATE Port 2 |
+        [ ATE Port 1 ] ----  |   DUT   |          |            |
+                             |         | --eBGP-- | ATE Port 3 |
+    ```
+
+*   Traffic is generated from ATE Port 1.
+*   ATE Port 2 is used as the destination port for PF. eBGP peerings
+        on this port announces BG4IE NH.
+*   ATE Port 3 is used as the fallback destination when PBR NH routes
```
Comment (dplore) [inline feature/policy_forwarding/otg_tests/match_dscp_indirect_next_hop/README.md:28]: Should stick to using  PF  to refer to policy-forwarding.  

```suggestion
*   ATE Port 3 is used as the fallback destination when PF NH routes
```

**Review by dplore — APPROVED** [review]

**Pull request merged**